recipes/entt/3.x.x/conanfile.py

```diff
@@ -55,15 +53,19 @@ def lazy_lt_semver(v1, v2):
             raise ConanInvalidConfiguration(
                 "%s requires a compiler that supports at least C++%s" % (self.name, minimal_cpp_standard))
 
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
     def source(self):
         get(self, **self.conan_data["sources"][self.version],
-            destination=self._source_subfolder, strip_root=True)
+            destination=self.source_folder, strip_root=True)
+
+    def build(self):
```

Comment:
Why is this method required?

Reply:
to avoid displaying a warning, but not mandatory